src/overlays/actors/ovl_En_Yabusame_Mark/z_en_yabusame_mark.c

```diff
@@ -69,7 +69,7 @@ typedef struct {
     /* 0x0C */ f32 unk_C;
 } unknownStruct; // size = 0x10
 
-unknownStruct D_80B4362C[] = {
+static unknownStruct D_80B4362C[] = {
```

Comment:
while youre at it, can you change the name of that struct to `YabasumeUnkStruct` or something, camel case is not correct for that